CMakeLists.txt

```diff
@@ -23,23 +23,19 @@ if(COMPILE_ESP)
                     COMPONENTS esp32 freertos esptool_py
                     SDKCONFIG ${CMAKE_BINARY_DIR}/sdkconfig
                     BUILD_DIR ${CMAKE_BINARY_DIR})
-    set(ALIAS_PREFIX "idf")
 else()
     message("Compiling locally")
-    set(ALIAS_PREFIX "SwarmUs")
     set(ENABLE_TESTS ON)
 endif()
 
-# Include sources
-add_subdirectory(src)
 
 #set executable name
 set(elf_file ${CMAKE_PROJECT_NAME}.elf)
```

Comment:
Not sure if we should leave the .elf even for the POSIX build. @xgroleau what do you think?

Reply:
Yea we could remove the .elf extension. The user knows it's a target build anyway. If you REALLY want the .elf, you could use a generator like this:

add_executable(${CMAKE_PROJECT_NAME} $<$<BOOL:${COMPILE_ESP}>:".elf">
      src/main.c
)

Reply:
Done. I will keep the .elf because some esp-idf targets depend on it.